jupyterhub/values.yaml

```diff
@@ -50,12 +50,13 @@ hub:
   imagePullPolicy: IfNotPresent
   pdb:
     enabled: true
+    minAvailable: 1
```

Comment:
Perhaps we should add a note that a change here will require a `--force` update on help. You can verify by attempting to `kubectl edit pdb hub` and modify the value. If it is okay then we are good, but if not lets add a note warning about the need for a `--force` on next upgrade.

For the other pdb's as well if so.